x-pack/plugins/rollup/public/index_pattern_creation/rollup_index_pattern_creation_config.js

```diff
@@ -29,6 +30,9 @@ const rollupIndexPatternTooManyMatchesError = i18n.translate('xpack.rollupJobs.e
 const rollupIndexPatternIndexLabel = i18n.translate('xpack.rollupJobs.editRollupIndexPattern.createIndex.indexLabel',
   { defaultMessage: 'Rollup' });
 
+const rollupIndexPatternIndicesErrorTitle = i18n.translate('xpack.rollupJobs.editRollupIndexPattern.createIndex.indicesErrorTitle',
```

Comment:
It's been suggested to me on other PRs that we not do the translations statically, but rather do them inside functions, so maybe do a getRollupIndexPatternIndicesErrorTitle function that returns this?

Reply:
Maybe it'd be best if the localization team updates the [guidelines](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md) first and then we do a sweep of the codebase and make this change at once?